pynest/nest/lib/hl_api_info.py

```diff
@@ -371,7 +371,7 @@ def GetStatus(nodes, keys=None, output=''):
         raise TypeError("The first input (nodes) must be NodeCollection or a SynapseCollection with connection handles")
 
     if len(nodes) == 0:
-        return nodes
+        return
```

Comment:
`GetStatus()` always returns a tuple (array for JSON output). The consistent return value for an empty node collection would then be an empty tuple for default output and an empty JSON array for JSON output.